README.md

```diff
@@ -2,4 +2,52 @@
 
 ## Build
 
-TAG=master make
+TAG=v1.0-rc1 make
+
+## Build it manually
+
+### 1. Checkout aks-operator repository
+
+git checkout https://github.com/mjura/aks-operator
+
+
+### 2. Download required modules and build project
+
+cd aks-operator
+
+go build -o aks-operator main.go
+
+### 3. Deploy CRD AKSClusterConfig on K8s
+
+export KUBECONFIG=<kubeconfig_path>
+
+kubectl apply -f crds/aksclusterconfig.yaml
+
+
+### 4. Edit AKSClusterConfig setup
+
+cp examples/create-example.yaml examples/create-aks.yaml
+
+vim examples/create-aks.yaml
+
+### 5. Create AKS secret with clientID and clientSecret
+
+export REPLACE_WITH_K8S_SECRETS_NAME=aks-secret
+
+kubectl create secret generic $REPLACE_WITH_K8S_SECRETS_NAME --from-literal=clientId=<REPLACE_WITH_CLIENTID> --from-literal=clientSecret=<REPLACE_WITH_CLIENTSECRET>
+
+### 6. Start aks-operator
+
+./aks-operator 
+
+### 7. Apply AKSClusterConfig
+
+kubectl apply -f examples/create-aks.yaml
+
+### 8. Watch the logs
+
+### 9. To delete AKS cluster
+
+kubectl delete -f examples/create-aks.yaml
+
+
```

Comment:
Thanks for creating a README! We should wrap all commands with "\`" or "\`\`\`bash" characters for monospace readability. Perhaps, we should add a description at the top as well.

Reply:
Thank you @nickgerace I have added quote marks